src/SSA_stepper.jl

```diff
@@ -11,28 +11,43 @@ Highly efficient integrator for pure jump problems that involve only
 
 ## Examples
 SIR model:
-```julia
+```jldoctest; output = false
 using DiffEqJump
 β = 0.1 / 1000.0; ν = .01;
 p = (β,ν)
-rate1(u,p,t) = p[1]*u[1]*u[2]  # β*S*I
+
+function rate1(u,p,t) 
+    p[1]*u[1]*u[2]  # β*S*I
+end
+
 function affect1!(integrator)
   integrator.u[1] -= 1         # S -> S - 1
   integrator.u[2] += 1         # I -> I + 1
 end
+
 jump = ConstantRateJump(rate1,affect1!)
 
-rate2(u,p,t) = p[2]*u[2]      # ν*I
+function rate2(u,p,t)
+    p[2]*u[2]      # ν*I
+end
+
 function affect2!(integrator)
   integrator.u[2] -= 1        # I -> I - 1
   integrator.u[3] += 1        # R -> R + 1
 end
+
 jump2 = ConstantRateJump(rate2,affect2!)
+
 u₀    = [999,1,0]
 tspan = (0.0,250.0)
 prob = DiscreteProblem(u₀, tspan, p)
 jump_prob = JumpProblem(prob, Direct(), jump, jump2)
 sol = solve(jump_prob, SSAStepper())
+
+sol isa ODESolution
```

Comment:
This is a good question. Let me know what you prefer. So the test will check the value of the final statement against a specified output. So if I just leave it as `sol = solve(...)` then the test will evaluate where the result is `RegularJump{true, typeof(regular_rate), typeof(regular_c), Nothing}(regular_rate, regular_c, 2, nothing)`. I was thinking that this comparison makes the test rather fragile, because if someone adjusts the internals of the type, this will make the tests fail. So that is the only reason that I did that, because `isa` will just return `RegularJump`, as opposed to the whole output :). 

I agree it is a little weird to see that test `sol is RegularJump`. I will check with the Documenter.jl folks and see if there is a way to just check if the code runs.

Reply:
Yeah let's drop the `isa` check then. I'd say if there is a way to just check a test runs without erroring let's do that, otherwise let's just keep these as normal julia blocks without the `isa` check line.

Reply:
I figured out some of the issue with the doctesting commands. So most people are using these `@example` blocks, but those only work in markdown documents. Those are run and evaluated when the docs are built. But the `@example` blocks don't work inside of docstrings. So that is where the challenge is. In docstrings, there is a need to compare the actual output of the code to the expected output. 
So I am trying to see if there is a smoother way to get this to work.